modules/nodes-pods-priority-preempt-about.adoc

```diff
@@ -17,6 +17,13 @@ Preemption does not necessarily remove all lower-priority pods from a node. The
 
 The scheduler considers a node for pod preemption only if the pending pod can be scheduled on the node.
 
+[id="non-preempting-priority-class_{context}"]
+== Non-preempting priority class ==
+
+Pods with PreemptionPolicy: Never will be placed in the scheduling queue ahead of lower-priority pods, but they cannot preempt other pods. A non-preempting pod waiting to be scheduled will stay in the scheduling queue, until sufficient resources are free, and it can be scheduled. Non-preempting pods, like other pods, are subject to scheduler back-off. This means that if the scheduler tries these pods and they cannot be scheduled, they will be retried with lower frequency, allowing other pods with lower priority to be scheduled before them.
```

Comment:
A few minor updates:

```
Pods with the preemption policy set to `Never` are placed in the scheduling queue ahead of lower-priority pods, but they cannot preempt other pods. A non-preempting pod waiting to be scheduled stays in the scheduling queue until sufficient resources are free and it can be scheduled. Non-preempting pods, like other pods, are subject to scheduler back-off. This means that if the scheduler tries these pods and they cannot be scheduled, they are retried with lower frequency, allowing other pods with lower priority to be scheduled before them.
```